doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -54,6 +57,287 @@ Additionally, maintaining ABI compatibility allows the nRF54H20 SoC binary compo
+* Improved stability.
+
+Removed
+-------
+
+* Temperature subscription
+
+IronSide Secure Element (IronSide SE) v22.1.0+13
+================================================
+
+Added
+-----
+
+* NCSDK-33486: Support for configuring TDD (CoreSight++) from local domains.
+
+IronSide Secure Element (IronSide SE) v22.0.4+12
+================================================
+
+Fixed
+-----
+
+* An issue where the device became stuck in recovery mode after performing a recovery upgrade (NCSDK-34258).
+* An issue where the eraseall operation, on a device in LCS ROT, was permitted regardless of the contents of UICR (NCSDK-34232).
+* An issue where the recovery firmware would incorrectly determine that UICR was corrupted (NCSDK-32241).
+
+Updated
+-------
+
+* sysctrl to 5.0.1: stability improvements.
+
+IronSide Secure Element (IronSide SE) v22.0.3+11
+================================================
+
+Fixed
+-----
+
+* psa_crypto:
+
+   * Bytes written are now correctly returned (in place of buffer size) (NCSDK-34172).
+   * Added missing ECC_MONTGOMERY_255 configuration (NCSDK-34200).
+   * Passing 0-sized buffers are now allowed for optional arguments (NCSDK-34171).
+
+* The default owner ID of some peripherals, where previously it was set to SECURE or SYSCTRL instead of APPLICATION (NCSDK-34187).
+
+IronSide Secure Element (IronSide SE) v22.0.2+10
+================================================
+
+Fixed
+-----
+
+* Missing CIPHER support in the PSA crypto service.
+
+IronSide Secure Element (IronSide SE) v22.0.1+9
+===============================================
+
+Updated
+-------
+
+* No functional change.
+  Version bump to verify update with live versions.
+
+IronSide Secure Element (IronSide SE) v22.0.0+8
+===============================================
+
+Added
+-----
+
+* This release is now signed with Nordic keys.
+  The SWD connection is still required to update IronSide using official tools.
+  For more information, run ``west ncs-ironside-se-update --help``.
+  A backward LCS transition is not required to update IronSide.
+* Added support for ``UICR.PROTECTEDMEM``, which enables integrity checking of an immutable bootloader.
+
+Updated
+-------
+
+* Increased the size of USLOT (IronSide + sysctrl) to 120 kB.
+* Increased the size of RSLOT (IronSide recovery firmware) to 20 kB.
+* Enabled downgrade protection for IronSide in debug builds.
+* Changed the owner ID used in the default global domain SPU configurations from ``NONE`` to ``APPLICATION``.
+  This means that all peripherals and split-ownership registers are accessible by the application core, PPR and FLPR by default.
+  Use ``UICR.PERIPHCONF`` to grant the radio core access to global domain peripherals.
+
+IronSide Secure Element (IronSide SE) v21.1.1
+=============================================
+
+Updated
+-------
+
+* CHIDX values are no longer required to be set when configuring ``PPIB_SUBSCRIBE_SEND`` or ``PPIB_PUBLISH_RECEIVE`` through ``PERIPHCONF``.
```

Comment:
```suggestion
Updated to not require CHIDX values to be set when configuring ``PPIB_SUBSCRIBE_SEND`` or ``PPIB_PUBLISH_RECEIVE`` through ``PERIPHCONF``.
```
No need for the subheadings when there is only one item below. Fix this in all similar cases.